js/index.js

```diff
@@ -194,60 +185,10 @@ $('#device').on('change', function() {
 });
 
 $(function () {
-    $('[data-toggle="tooltip"]').tooltip();
-    $('[data-toggle="tooltip"]').tooltip({
-        trigger: "manual"
-    });
-    
-    $('[data-toggle="tooltip"]').on('mouseleave', function () {
-        $(this).tooltip('hide');
-    });
-    
-    $('[data-toggle="tooltip"]').on('mouseenter', function () {
-        $(this).tooltip('show');
-    });
-    
-    $('[data-toggle="tooltip"]').on('click', function () {
-        $(this).tooltip('hide');
-    });
+    utilities.initializeTooltips();
 })
 
-function convertUint8ArrayToBinaryString(u8Array) {
-	var i, len = u8Array.length, b_str = "";
-	for (i=0; i<len; i++) {
-		b_str += String.fromCharCode(u8Array[i]);
-	}
-	return b_str;
-}
-
-function convertBinaryStringToUint8Array(bStr) {
-	var i, len = bStr.length, u8_array = new Uint8Array(len);
-	for (var i = 0; i < len; i++) {
-		u8_array[i] = bStr.charCodeAt(i);
-	}
-	return u8_array;
-}
-
-function handleFileSelect(evt) {
-    var file = evt.target.files[0];
-    var reader = new FileReader();
-
-    reader.onload = (function(theFile) {
-        return function(e) {
-            file1 = e.target.result;
-            evt.target.data = file1;
-        };
-    })(file);
-
-    reader.readAsBinaryString(file);
-}
-
-
-document.getElementById('selectFile1').addEventListener('change', handleFileSelect, false);
-
-function _sleep(ms) {
-    return new Promise(resolve => setTimeout(resolve, ms));
-}
+document.getElementById('selectFile1').addEventListener('change', utilities.handleFileSelect, false);
```

Comment:
The third parameter in addEventListener method anyway defaults to false, right? Just an observation, its okay even if we don't change. But the third parameter specifies if the events capture is required down the DOM tree.